cypress/support/pages/home.pageObject.js

```diff
@@ -7,6 +7,21 @@ class HomePageObject extends PageObject {
     return cy.getByDataCy('username-link');
   }
 
+  getByDataAttr (dataAttr) {
+   return cy.get(`[data-cy="${dataAttr}"]`);
+ }
+
+ fillField(dataAttr, newData) {
+   this.getByDataAttr(dataAttr)
+     .clear()
+     .type(newData);
+ }
+
+ fillFields(dataAttr, newData) {
+   this.getByDataAttr(dataAttr)
+     .type(newData);
```

Comment:
what is the difference between those 2 methods?)
```suggestion
 fillField(dataAttr, newData) {
   this.getByDataAttr(dataAttr)
     .clear()
     .type(newData);
 }

 fillFields(dataAttr, newData) {
   this.getByDataAttr(dataAttr)
     .type(newData);
```